CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Some examples, more below in the actual changelog (newer entries are more likely
 ### Added
 * vsphere/v1: Template bindings and FindNamedTemplate helper added to retrieve templates by name (#148, @marioreggiori)
 * generic client: mock implementation (#139, @marioreggiori)
+* kubernetes/v1: implement generic client bindings for cluster and node_pool resources (#151, @marioreggiori)
```

Comment:
I'd favor "kubernetes/v1: add Cluster and NodePool bindings", but that's a nitpick ^^